src/main/kotlin/no/nav/tilleggsstonader/sak/statistikk/vedtak/VedtaksstatistikkDvh.kt

```diff
@@ -22,49 +26,74 @@ class VedtaksstatistikkDvh(
     val eksternBehandlingId: Long,
     val relatertBehandlingId: Long?, // Ekstern behandlingsid på relatert behandling
     val adressebeskyttelse: AdressebeskyttelseDvh,
-    val tidspunktVedtak: ZonedDateTime,
+    val tidspunktVedtak: LocalDateTime,
     val målgruppe: List<MålgruppeDvh>,
     val aktivitet: List<AktivitetDvh>,
-    val vilkårsvureringer: List<VilkårsvurderingDvh>,
+    val vilkårsvurderinger: List<VilkårsvurderingDvh>,
     val person: String,
     val barn: List<BarnDvh>,
     val behandlingType: BehandlingTypeDvh,
     val behandlingÅrsak: BehandlingÅrsakDvh,
     val vedtakResultat: VedtakResultatDvh,
-    val vedtaksperioder: VedtaksperioderDvh,
+    val vedtaksperioder: List<VedtaksperiodeDvh>,
     val utbetalinger: List<UtbetalingDvh>,
     val stønadstype: StønadstypeDvh = BARNETILSYN,
-    val kravMottatt: LocalDate,
+    val kravMottatt: LocalDate?,
     val årsakRevurdering: ÅrsakRevurderingDvh? = null,
     val avslagÅrsak: AvslagÅrsakDvh? = null,
 
-)
+    )
 
 enum class StønadstypeDvh {
     BARNETILSYN,
 }
 
 data class UtbetalingDvh(
-    val gyldigFom: LocalDate,
-    val gyldigTom: LocalDate,
+    val utbetalingsdato: LocalDate,
     val beløp: Int,
-    val satstype: SatstypeDvh = SatstypeDvh.DAGLIG,
-)
+) {
+    companion object {
+        fun fraDomene(ytelser: List<AndelTilkjentYtelse>) = ytelser.map {
+            UtbetalingDvh(utbetalingsdato = it.fom, beløp = it.beløp)
+        }
+    }
+}
 
 enum class SatstypeDvh {
     DAGLIG,
 }
 
-data class VedtaksperioderDvh(
+data class VedtaksperiodeDvh(
     val fomDato: LocalDate,
     val tomDato: LocalDate,
     val utgifter: Int,
-)
+) {
+    companion object {
+        // TODO: Map fra faktiske vedtaksperioder når vi har det (også relatert til revurdering)
+
+        fun fraDomene(ytelser: List<AndelTilkjentYtelse>) = ytelser.map {
```

Comment:
Ref https://github.com/navikt/tilleggsstonader-sak/pull/302/files#r1590953497
```suggestion
        fun fraDomene(ytelser: List<Stønadsperiode>) = ytelser.map {
```

Reply:
Stønadsperioder er egentlig heller ikke riktig her ettersom det som faktisk er vedtaksperioder er snittet mellom stønadsperioder og utgiftsperioder, som aldri lagres ned